app-tests/docker-compose-app-tests-openfga.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+services:
+  broadcast_channel:
+    image: postgres:alpine
+    environment:
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    networks:
+      - opal-network
+
+  opal_server:
+    image: permitio/opal-server:latest
+    environment:
+      - OPAL_BROADCAST_URI=postgres://postgres:postgres@broadcast_channel:5432/postgres
+      - UVICORN_NUM_WORKERS=4
+      - OPAL_POLICY_REPO_URL=https://github.com/daveads/opal-example-policy-openfga
```

Comment:
@daveads here also, needs to be modified to permitio or a var GITHUB_ACCOUNT, ok?

Reply:
oh i this can be taken care of after merge... right ?